grpc/public/password_reset.go

```diff
@@ -0,0 +1,32 @@
+package public
+
+import (
+	"github.com/keratin/authn-server/app"
+	"github.com/keratin/authn-server/app/services"
+	authnpb "github.com/keratin/authn-server/grpc"
+	context "golang.org/x/net/context"
+)
+
+var _ authnpb.PasswordResetServiceServer = passwordResetServer{}
+
+type passwordResetServer struct {
+	app *app.App
+}
+
+func (s passwordResetServer) RequestPasswordReset(ctx context.Context, req *authnpb.PasswordResetRequest) (*authnpb.PasswordResetResponse, error) {
+
+	account, err := s.app.AccountStore.FindByUsername(req.GetUsername())
+	if err != nil {
+		panic(err)
```

Comment:
Should this really be fatal? What if my `AccountStore` is temporarily unavaillable?